Makefile

```diff
@@ -221,13 +221,14 @@ fetch: $(GIT_UG_REPO_PATH) $(GIT_SAMPLE_DATA_REPO_PATH) $(GIT_TEST_DATA_REPO_PAT
 
 # Python conda environment management
 env:
-		$(PYTHON) ./scripts/convert-requirements-to-conda-yml.py requirements.txt requirements-dev.txt requirements-gui.txt > requirements-all.yml
-		$(CONDA) create -p $(ENV) -y -c conda-forge python=3.8 nomkl
-		$(CONDA) env update -p $(ENV) --file requirements-all.yml
-		@echo "----------------------------"
-		@echo "To finish the conda env install:"
-		@echo ">> conda activate ./$(ENV)"
-		@echo ">> make install"
+	@echo "NOTE: requires 'requests' be installed in base Python"
+	$(PYTHON) ./scripts/convert-requirements-to-conda-yml.py requirements.txt requirements-dev.txt requirements-gui.txt > requirements-all.yml
+	$(CONDA) create -p $(ENV) -y -c conda-forge python=3.8 nomkl
+	$(CONDA) env update -p $(ENV) --file requirements-all.yml
+	@echo "----------------------------"
+	@echo "To finish the conda env install:"
```

Comment:
May I suggest changing this to "To activate the new conda environment and install natcap.invest:"

As-is it kinda implies there are still steps left to finish the `make env` process.